src/styles/main.scss

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
+}
+
+$font-family-days: Arial, Helvetica, sans-serif;
+$font-size-days: 30px;
+$color-standart: #eee;
+$color-hover: #ffbfcb;
+$box-size: 100px;
+$callendar-width: 726px;
+$box-gap: 1px;
+$days: (
+  'mon': 0,
+  'tue': 101,
+  'wed': 202,
+  'thu': 303,
+  'fri': 404,
+  'sat': 505,
+  'sun': 606,
+);
```

Comment:
Please, create a variables file, and move it to this file. 
Thank import inside the main file. 
Thanks)